packages/text-input/index.js

```diff
@@ -1,42 +1,48 @@
+import classNames from 'classnames'
 import { v1 as uuidv1 } from 'uuid'
+import s from './style.module.css'
 
-function TextInput(props) {
-  //  `field` and `form` props come from Formik
-  //  https://jaredpalmer.com/formik/docs/api/field
-  const { field, form, type, label, placeholder, theme } = props
```

Comment:
Moved prop de-structuring into the function argument area, to align with our approach in other components.